docs/hello-world.ftd

```diff
@@ -9,4 +9,8 @@ Hi My name is Abrar Khan
 Hi My name is Abrar Khan
 
 
-Hello world testing
\ No newline at end of file
+Hello world testing
+
+
+
+
```

Comment:
Delete this entire folder. Whatever you need for testing move to a folder inside tests folder.